proprietary/design-tokens/src/component/utrecht/link.tokens.json

```diff
@@ -2,6 +2,7 @@
   "utrecht": {
     "link": {
       "color": { "value": "{utrecht.color.blue.35}" },
+      "link-icon-size": { "value": "1.2em" },
```

Comment:
Let's nest it instead: `{ "utrecht": { "link": { "icon": { "size": { "value": "..."`